### ISSUE TITLE
resolution of namespace in syntax quote resolved to wrong namespace

### ISSUE THREAD

**Issue by borkdude** [babashka/nbb#180]
```
user=> (ns foo)
#object[In foo]
foo=> (defn x[])
#'foo/x
foo=> `x
user/x
```

**Comment by borkdude** [comment]
This turns out only to be a REPL problem

```
$ node lib/nbb_main.js -e "(ns foo) (defn x []) \`x"
foo/x
```

**Comment by borkdude** [comment]
Fixed in 0.3.11
